CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.4.0 (2019-10-09)
+
```

Comment:
I'm gonna nerd out for a moment, but when used as an adjective, it should be full-screen. In this scenario, it's being used as an adverb though. 

I think we could say "from displaying in full screen." and then call that good.

Reply:
Also, shouldn't this be 2 separate changelog entries? Just based on the descriptions, it seems like 2 unrelated changes in the same PR. I'm pretty sure that in that scenario, we usually have separate entries.

Reply:
They're related in that the input property needed to be added to popovers to allow the dropdown component to leverage it. You guys can tell me whatever you think is best here.

Reply:
I think it's worth calling out that info to make it feel less random. Maybe something like?

Added the `allowFullscreen` input property to the popover component. The dropdown component uses the `allowFullscreen` input property to prevent dropdown menu items from displaying in full screen.

Reply:
Yeah I could get behind that!